.github/workflows/labeler.yml

```
@@ -0,0 +1,19 @@
+# Add labels to pull request
+# uses logic in .github/labeler.yml
+
+---
+name: "Add labels to pull request"
+
+on:
+- pull_request_target
+
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v4
```

we may also need to skip dependabot PRs, as those are done per-branch.

I'm guessing... ?

```suggestion
    - uses: actions/labeler@v4
      if: ${{ github.event.pull_request.user.login }} != 'dependabot[bot]'
```

---

This syntax worked in my testing, thanks!